app/javascript/components/collections/landing/Pagination.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+
+class Pagination extends Component {
+  constructor(props) {
```

Comment:
Don't need this `constructor` function.  And looks like this component could be a standard functional component, as it doesn't handle any state directly.  Not a big deal, just a little cleaner.